homework.md

```diff
@@ -4,110 +4,109 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34 ?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Used to make website style.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used for developing dynamic websites.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+When people want to share,manage and version control of their code they can use Git & Github.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You take a copy of that repository to your profile.
 
-## Question 10 
+## Question 10
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You take a copy of that repository to your computer.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+A pull request is an event in Git where a contributor asks a maintainer of a Git repository to review code they want to merge into a project.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+Git log is a utility tool to review and read a history of everything that happens to a repository.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+When you push changes, you send the committed changes in your local repository to the remote repository on GitHub.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+Github is a web-based platform used for version control.A place for us to store and share our code.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send you code changes to Github.
```

Comment:
A 'commit' saves a change that you have made to your **local repository** (on your laptop) using git. A 'push' is what sends our code changes to the **GitHub repository** (online)

Reply:
Thanks Ellie. Your answer is much better and clear.